test/integrations/destinations/gainsight_px/network.ts

```diff
@@ -1,222 +1,514 @@
+      config: {
+        transitional: {
+          silentJSONParsing: true,
+          forcedJSONParsing: true,
+          clarifyTimeoutError: false,
         },
-        httpRes: {
-            data: {
-                externalapierror: {
-                    status: "NOT_FOUND",
-                    message: "User was not found for parameters {id=absent-id}",
-                    debugMessage: null,
-                    subErrors: null
-                }
-            },
-            status: 404
+        adapter: ['xhr', 'http'],
+        transformRequest: [null],
+        transformResponse: [null],
+        timeout: 60000,
+        xsrfCookieName: 'XSRF-TOKEN',
+        xsrfHeaderName: 'X-XSRF-TOKEN',
+        maxContentLength: 100000000,
+        maxBodyLength: 100000000,
+        env: {},
+        headers: {
+          Accept: 'application/json, text/plain, */*',
+          'Content-Type': 'application/json',
+          'X-APTRINSIC-API-KEY': 'sample-api-key',
+          'User-Agent': 'axios/1.6.5',
+          'Accept-Encoding': 'gzip, compress, deflate, br',
         },
-    },
-    {
-        httpReq: {
-            url: 'https://api.aptrinsic.com/v1/users/stanley-kubrick',
-            method: 'GET',
+        method: 'get',
+        withCredentials: false,
+        responseType: 'json',
+        responseEncoding: 'utf8',
+        maxRedirects: 5,
+        httpAgent: {
+          _events: {},
+          _eventsCount: 2,
+          defaultPort: 80,
+          protocol: 'http:',
+          options: { keepAlive: true, noDelay: true, path: null },
+          requests: {},
+          sockets: {},
+          freeSockets: {},
+          keepAliveMsecs: 1000,
+          keepAlive: true,
+          maxSockets: null,
+          maxFreeSockets: 256,
+          scheduling: 'lifo',
+          maxTotalSockets: null,
+          totalSocketCount: 0,
         },
-        httpRes: {
-            data: {
-                "id": "ecorp-id",
-                "name": "ECorp",
-                "trackedSubscriptionId": "",
-                "sfdcId": "",
-                "lastSeenDate": 0,
-                "dunsNumber": "",
-                "industry": "software",
-                "numberOfEmployees": 400,
-                "sicCode": "",
-                "website": "www.ecorp.com",
-                "naicsCode": "",
-                "plan": "premium",
-                "location": {
-                    "countryName": "",
-                    "countryCode": "",
-                    "stateName": "",
-                    "stateCode": "",
-                    "city": "",
-                    "street": "",
-                    "postalCode": "",
-                    "continent": "",
-                    "regionName": "",
-                    "timeZone": "",
-                    "coordinates": {
-                        "latitude": 0.0,
-                        "longitude": 0.0
-                    }
+        httpsAgent: {
```

Comment:
do we need to keep the whole thing?

Reply:
It was only included to suggest that we got an AxiosError that was very long.

Reply:
Got it but do we need to test any code functionality?

Reply:
Yes, in a way. So we saw in our analysis that we had a keyword `upgrade required`, which is why the score of version deprecation increased leading to false-positive alerts.
In the bytes below, I think we would have some information about the keyword I was talking about hence put the information explicitly here

Through the test-case we are making sure that we don't put such information into payload when there is a failure from destination(that leads to `AxiosError`)